sample-resource-plugin/build.gradle

```diff
@@ -77,7 +77,7 @@ configurations {
 
             // Duplicate dependencies with conflicting versions from org.opensearch.plugin:transport-grpc
             // Force versions present in security plugin
-            force "com.google.errorprone:error_prone_annotations:2.44.0"
+            force "com.google.errorprone:error_prone_annotations:2.45.0"
```

Comment:
I think we should refactor this to refer to the version from core's version catalog now.